--- config_old.py	2021-12-08 12:57:17.401154090 +0300
+++ configuration.py	2021-12-08 12:56:32.812466368 +0300
@@ -1,9 +1,7 @@
-# coding: utf-8
-
 """
-    Preprocessor API
-
-    This is MIPT-2PC preprocessor API  # noqa: E501
+    User exchange API
+    # Patched by ProValdi
+    User exchange API  # noqa: E501
 
     OpenAPI spec version: 1.0.0
     Contact: prokhorov.va@phystech.edu
@@ -21,6 +19,7 @@
 import six
 from six.moves import http_client as httplib
 
+import os
 
 class TypeWithDefault(type):
     def __init__(cls, name, bases, dct):
@@ -46,7 +45,11 @@
     def __init__(self):
         """Constructor"""
         # Default Base url
-        self.host = "http://localhost:8080/MIPT-2PC/preprocessor/1.0.0"
+        # Default Base url, CLIENT_A = 8080, CLIENT_B = 8081, NEIGHBOR_PORT = CURRENT_CLIENT_A? CLIENT_B:CLIENT_A
+        if os.getenv("CLIENT_A", None) is not None:
+            self.host = "http://localhost:" + os.getenv("NEIGHBOR_PORT") + "/MIPT-2PC/user/1.0.0"
+        if os.getenv("CLIENT_B", None) is not None:
+            self.host = "http://localhost:" + os.getenv("NEIGHBOR_PORT") + "/MIPT-2PC/user/1.0.0"
         # Temp file folder for downloading files
         self.temp_folder_path = None
 
@@ -241,4 +244,4 @@
                "Python Version: {pyversion}\n"\
                "Version of the API: 1.0.0\n"\
                "SDK Package Version: 1.0.0".\
-               format(env=sys.platform, pyversion=sys.version)
\ В конце файла нет новой строки
+               format(env=sys.platform, pyversion=sys.version)
