--- __main__old.py	2021-12-08 00:06:21.309197804 +0300
+++ __main__.py	2021-12-08 00:06:48.848851253 +0300
@@ -1,6 +1,8 @@
 #!/usr/bin/env python3
+# patched by ProValdi
 
 import connexion
+import os
 
 from swagger_server import encoder
 
@@ -9,7 +11,11 @@
     app = connexion.App(__name__, specification_dir='./swagger/')
     app.app.json_encoder = encoder.JSONEncoder
     app.add_api('swagger.yaml', arguments={'title': 'User exchange API'}, pythonic_params=True)
-    app.run(port=8080)
+    if os.getenv('CLIENT_A', None) is not None:
+        cliPort = os.getenv('CLIENT_A', None)
+    if os.getenv('CLIENT_B', None) is not None:
+        cliPort = os.getenv('CLIENT_B', None)
+    app.run(port=int(cliPort))
 
 
 if __name__ == '__main__':
